run/SU2_preCICE_CHT.py

```diff
@@ -26,240 +26,247 @@
 
 def main():
 
-  # Command line options
-  parser=OptionParser()
-  parser.add_option("-f", "--file", dest="filename", help="Read config from FILE", metavar="FILE")
-  parser.add_option("--parallel", action="store_true",
-                    help="Specify if we need to initialize MPI", dest="with_MPI", default=False)
-
-  # preCICE options with default settings
-  parser.add_option("-p", "--precice-participant", dest="precice_name", help="Specify preCICE participant name", default="Fluid" )
-  parser.add_option("-c", "--precice-config", dest="precice_config", help="Specify preCICE config file", default="../precice-config.xml")
-  parser.add_option("-m", "--precice-mesh", dest="precice_mesh", help="Specify the preCICE mesh name", default="Fluid-Mesh")
-  parser.add_option("-r", "--precice-reverse", action="store_true", dest="precice_reverse", help="Include flag to have SU2 write temperature, read heat flux", default=False)
+	# Command line options
```

Comment:
For the whitespace changes, I guess we need an automated formatting solution.
This could happen in a separate PR. Ideally, the conversion to tabs should not happen in this PR either, as these are two different things: it will be a bit difficult to see the fixes in the future.